web/modules/custom/feedback/src/Form/FeedbackForm.php

```diff
@@ -0,0 +1,227 @@
+<?php
+
+namespace Drupal\feedback\Form;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Database\Connection;
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Session\AccountProxyInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+/**
+ * Provides an feedback form.
+ */
+class FeedbackForm extends FormBase {
+
+  /**
+   * The Database Connection.
+   *
+   * @var \Drupal\Core\Database\Connection
+   */
+  protected $database;
+
+  /**
+   * Retrieves the request stack.
+   *
+   * @var \Symfony\Component\HttpFoundation\RequestStack
+   */
+  protected $requestStack;
+
+  /**
+   * Gets the current active user.
+   *
+   * @var \Drupal\Core\Session\AccountProxyInterface
+   */
+  protected $currentUser;
+
+  /**
+   * Retrieves the configuration factory.
+   *
+   * @var \Drupal\Core\Config\ConfigFactoryInterface
+   */
+  protected $configFactory;
+
+  /**
+   * FeedbackForm constructor.
+   *
+   * @param \Drupal\Core\Database\Connection $database
+   *   The Database Connection.
+   * @param \Symfony\Component\HttpFoundation\RequestStack $request_stack
+   *   Retrieves the request stack.
+   * @param \Drupal\Core\Session\AccountProxyInterface $current_user
+   *   Gets the current active user.
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
+   *   Retrieves the configuration factory.
+   */
+  public function __construct(Connection $database, RequestStack $request_stack, AccountProxyInterface $current_user, ConfigFactoryInterface $config_factory) {
+    $this->database = $database;
+    $this->requestStack = $request_stack;
+    $this->currentUser = $current_user;
+    $this->configFactory = $config_factory;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('database'),
+      $container->get('request_stack'),
+      $container->get('current_user'),
+      $container->get('config.factory')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'feedback_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    // Check GET value.
+    $getFilterValue = $this->requestStack->getCurrentRequest()->query->get('fid');
+    // Checking for form editing.
+    $record = array();
+    if (isset($getFilterValue)) {
+      $query = $this->database->select('feedback', 'f')
```

Comment:
it's not a correct way to load entity

use entityTypeManager->getStorage('feedback')->load()

also I don't think you have to load something - if you are using entity system correct way – entities will be loaded automatically

just check other entities implementation in core or contrib modules